light_curves/light_curve_generator.md

```diff
@@ -372,30 +359,14 @@ callback = parallel_df_lc.append  # will be called once on the result returned b
 with mp.Pool(processes=n_workers) as pool:
 
     # start the processes that call the archives
-    pool.apply_async(
-        Gaia_get_lightcurve, (sample_table, 1/3600., 0), callback=callback
-    )
-    pool.apply_async(
-        HEASARC_get_lightcurves, (sample_table, heasarc_catalogs), callback=callback
-    )
-    pool.apply_async(
-        HCV_get_lightcurves, (sample_table, hcv_radius), callback=callback
-    )
-    pool.apply_async(
-        Icecube_get_lightcurve, (sample_table , 3), callback=callback
-    )
-    pool.apply_async(
-        Panstarrs_get_lightcurves, (sample_table, panstarrs_radius), callback=callback
-    )
-    pool.apply_async(
-        TESS_Kepler_get_lightcurves, (sample_table, lk_radius), callback=callback
-    )
-    pool.apply_async(
-        WISE_get_lightcurves, (sample_table,  wise_radius, bandlist), callback=callback
-    )
-    pool.apply_async(
-        ZTF_get_lightcurve, (sample_table, ztf_nworkers), callback=callback
-    )
+    pool.apply_async(gaia_get_lightcurves, args=(sample_table,), callback=callback)
```

Comment:
what is the rationale for hiding the arguments of the functions other than sample_table?  
I would have thought that some of these arguments the user might actually want to change, or at least know they could change, eg., heasarc_catalogs, so if we show the syntax in the tutorial, then it is more clear to the user what the free parameters are.

Reply:
Now that everything but `sample_table` has a default they aren't required here anymore, and the calls are simpler without them. The serial section shows what they are at how to use them (at least most of them). This section  gives one example (ZTF) of how to submit them in this call, and I figured it was fine to leave the others out and let the user add them if they want.

That was my rational. but I'm happy to add them back in if you want.

Reply:
I think we agreed to add redshift info back to the serial calls, but leave the parallel section as is until the review for PR #215